src/Hatchinq/Menu.elm

```diff
@@ -30,65 +29,26 @@ type alias State =
     }
 
 
-subscriptions : Config msg -> State -> (Message msg -> msg) -> Sub msg
-subscriptions config state lift =
+subscriptions : State -> (Message msg -> msg) -> Sub msg
+subscriptions state lift =
     if state.isOpen then
-        Browser.Events.onMouseDown (outsideTarget "menu")
-            |> Sub.map lift
-
+        Browser.Events.onMouseDown (outsideTarget "menu"
```

Comment:
indentation looks a bit funny here. You should use the `elm-format` tool which takes care of the alignment by its own.